lib/cpp/src/thrift/windows/config.h

```diff
@@ -36,6 +36,9 @@
 #define USE_BOOST_THREAD 1
 #endif
 
+// Something that defines PRId64 is required to build
+#define HAVE_INTTYPES_H 1
+
```

Comment:
it's probably left-over from pre-cmake times: vstudio project included in repo did not generate any config.h, yet something had to be included...